src/components/EnrolledLearnersForInactiveCoursesTable/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import TableContainer from '../../containers/TableContainer';
+import { formatTimestamp } from '../../utils';
+import EnterpriseDataApiService from '../../data/services/EnterpriseDataApiService';
+
+const EnrolledLearnersForInactiveCoursesTable = () => {
+  const tableColumns = [
+    {
+      label: 'Email',
+      key: 'user_email',
+      columnSortable: true,
+    },
+    {
+      label: 'Total Course Enrollment Count',
+      key: 'enrollment_count',
+      columnSortable: true,
+    },
+    {
+      label: 'Total Completed Courses Count',
+      key: 'course_completion_count',
+      columnSortable: true,
+    },
+    {
+      label: 'Last Activity Date',
+      key: 'last_activity_date',
+      columnSortable: true,
+    },
+  ];
+
+  const formatLearnerData = learners => learners.map(learner => ({
```

Comment:
We rename it to `formatLastActivityDate` and move this to utils since this is also used in `PastWeekPassedLearnersTable` as well, may be some other place as well.

Reply:
@mushtaqak I think we should keep this function as `formatLearnerData` in case the columns we show in the table end up changing later on. If that happens, we would need to change this function to potentially format those additional columns.